### ISSUE TITLE
MultiSelect: Form elements must have labels accessibility violation

### ISSUE THREAD

**Issue by elena-ung** [primefaces/primeng#18183]
Install the Axe Dev Tools for Accessibility plug-in.
Navigate to https://v17.primeng.org/multiselect on Chrome browser
Scan the page with Axe

Observe 'Form elements should have labels' error reported for hidden input

![Image](https://github.com/user-attachments/assets/7bed3200-1e14-4fa4-83df-801e23dedab7)

Angular version: 18.2.9
PrimeNG version: v17
Node version: 18.19.1

**Comment by breakone** [comment]
@elena-ung I think, you can add an `ariaLabel` to prevent this issue:

```
<p-multiselect
  [ariaLabel]="'my label'"
  ...
```
